xsync/Cargo.toml

```diff
@@ -12,16 +12,16 @@ edition = "2021"
 
 [workspace.dependencies]
 anyhow = "1.0"
-cargo_toml = "0.20"
-cargo-lock = "10.0.0"
-clap = "4.2"
-dunce = "1.0.5"
```

Comment:
Don't remove patch version constraints. If this gets pushed to crates.io one day, Cargo.lock constraints don't apply but Cargo.toml ones do, and we don't really know if e.g. dunce 1.0.4 is acceptable.